### DISCUSSION TITLE
Custom theme is not reflecting in embedded form.

### DISCUSSION THREAD

**Discussion by sreehari-aot** [AOT-Technologies/forms-flow-ai#1207] [General]
Applied a custom theme for `formsflow-web` by making use of Formio API  `Formio.use(customtheme);` but it is working with formsflow application but not when the form is embedded.

**Comment by AbijithS-aot** [comment]
You may miss the import files inside the forms-flow-ai-web-components repo. You can use your theme in form embedding by following steps :

*  Clone the repo forms-flow-ai-web-components

               git clone https://github.com/AOT-Technologies/forms-flow-ai-web-components.git

*   install node modules

               npm install

*  install npm package for formio theming

              npm i forms-flow-formio-theme

*  import formsflowFormioTheme from the package.

              import formsflowFormioTheme from 'forms-flow-formio-theme'

*  Use our custom theming in formio

               Formio.use(formsflowFormioTheme)

*  Add following css link where we defined our custom css classes

    note :  Use the css links in the website which you are embedding the form.

                <link rel="stylesheet" href="https://forms-flow-ai-stylesheet.pages.dev/formsflow.min.css">     

                <link rel="stylesheet" href="https://forms-flow-ai-stylesheet.pages.dev/formsflow-theme.min.css">    

                <link rel="stylesheet" href="https://forms-flow-ai-stylesheet.pages.dev/formsflow-custom.css">

  note :

*  you can create a build by running the command
          
                npm run build

*  And you can host this build file as cdn and you can use that cdn for form embedding.
